drivers/Makefile.dep

```diff
@@ -148,6 +148,10 @@ ifneq (,$(filter ethos,$(USEMODULE)))
   USEMODULE += tsrb
 endif
 
+ifneq (,$(filter extend_dac,$(USEMODULE)))
```

Comment:
Might need ```FEATURES_REQUIRED += periph_dac``` here.

Reply:
I did't use it here by intention. If you require peripheral DAC functionality, only MCUs that already have DAC channels can use DAC extensions. But, especially MCUs that do not have DAC channels can benefit from the DAC extension.